bin/simple_git_deploy.sh

```diff
@@ -0,0 +1,113 @@
+#!/bin/bash
+
+################################################################################
+# Simple deploy script of a BEdita4 instance via git
+#
+# Usage: simple_git_deploy.sh [deploy_user] [app_user] [plugins_list]
+#
+#  * `deploy_user`: user responsible for git/deploy operations, typically file owner
+#  * `app_user`: user launching API actions, typically web server user
+#  * `plugins_list`: comma separated list of active plugins to update together with core
+#
+# Notes:
+#  - when no arguments are passed no deploy or app user is set and all plugins found in `plugins/` are updated
+#  - on single argument only `deploy_user` is set, and used as `app_user`` too
+#  - `plugins_list` use relative path inside `plugins/` for names, example: `BEdita/DevTools,MyPlugin`
+#     on empty `plugins list` all plugins found in `plugins/` are updated
+#
+################################################################################
+
+DEPLOY_USER=''
+DEPLOY_PREFIX_CMD=''
+APP_USER=''
+APP_PREFIX_CMD=''
+
+BE4_DIR=$PWD
```

Comment:
```bash
BE4_DIR=$(dirname $(cd $(dirname "$0") && pwd))
```

This apparently works, and removes the requirement of the working directory.

Reply:
However if you're not on the root folder other commands will fail....

Reply:
Uhm… Right. `git` and `composer` should include the `-C $BE4_DIR` option, plugins auto-list should be aware of the working dir, and `bin/cake` should become `$BE4_DIR/bin/cake`.

I haven't tried, but I think it should work… However this is not essential — 99% of the times the working directory will be the correct one.

Reply:
Done, now the script is callable from anywhere